app/controllers/static_controller.rb

```diff
@@ -10,7 +10,7 @@ def accept_terms_of_service
   end
 
   def not_found
-    render plain: "not found", status: :not_found
```

Comment:
I remember there being something surrounding error/404 templates and exceptions related to template rendering, but I think they might have been resolved at some point along with the changes to error handling. I'll make sure that this works as intended on my side.

Reply:
Do you mean what happenes when the template might have a syntax error or something?
`application_controller` has a generic `rescue_from` while it's missing from the static one. This shows the error stack and other stuff to the user. I don't know how it might look in production but I think it's probably the same?

Should I had a error handler to the static controller as well?

Reply:
The old behavior is gone now, so this should be safe. If anything crops up I'll address it as needed.